adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -1149,24 +1149,24 @@
             if (!type || type == 'default') {
                 return {
                     login: function(options) {
-                        window.location.href = kc.createLoginUrl(options);
+                        window.location.replace(kc.createLoginUrl(options));
                         return createPromise().promise;
                     },
 
                     logout: function(options) {
-                        window.location.href = kc.createLogoutUrl(options);
+                        window.location.replace(kc.createLogoutUrl(options));
                         return createPromise().promise;
                     },
 
                     register: function(options) {
-                        window.location.href = kc.createRegisterUrl(options);
+                        window.location.replace(kc.createRegisterUrl(options));
                         return createPromise().promise;
                     },
 
                     accountManagement : function() {
                         var accountUrl = kc.createAccountUrl();
                         if (typeof accountUrl !== 'undefined') {
-                            window.location.href = accountUrl;
+                            window.location.replace(accountUrl);
```

Comment:
Shouldn't use replace here as this is actually navigating to the account management console. The app should remain in the history so the user can go back